portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,7 +36,8 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  // Using '2' to indicate an invalid sentiment score 
+  private static final int INVALID_SENTIMENT_SCORE = 2;
```

Comment:
please add a blank line to separate the constant declaration from the method declaration.